lobster/report.py

```diff
@@ -20,6 +20,9 @@
 import os.path
 import json
 from collections import OrderedDict
+from dataclasses import dataclass
+
+# from dataclasses import dataclass
```

Comment:
Is this comment necessary? If not we should remove it

Reply:
Done